front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarStatusesItem.tsx

```diff
@@ -42,39 +42,41 @@ interface Props {
   connectDropTarget: any;
 }
 
-class FilterSidebarStatusesItem extends React.PureComponent<Props> {
-  render() {
-    const { status, active, onClick, connectDropTarget, isOver, canDrop } =
-      this.props;
-
-    return connectDropTarget(
-      <div>
-        <Menu.Item active={active || (isOver && canDrop)} onClick={onClick}>
-          <ItemWrapper>
-            <ColorIndicator color={status.attributes.color} />
-            <div>
-              <StatusText>
-                <T value={status.attributes.title_multiloc} />
-              </StatusText>
-              {get(status, 'attributes.transition_type') === 'automatic' && (
-                <StyledText>
-                  &nbsp;
-                  <FormattedMessage {...messages.automatic} />
-                </StyledText>
-              )}
-            </div>
-          </ItemWrapper>
-        </Menu.Item>
-      </div>
-    );
-  }
-}
+const FilterSidebarStatusesItem = ({
+  status,
+  active,
+  onClick,
+  connectDropTarget,
+  isOver,
+  canDrop,
+}: Props) => {
+  return connectDropTarget(
+    <div>
```

Comment:
No need for this div?

Reply:
Not sure, just refactored to functional here. Not touching other stuff (thinking of List where removing a wrapper can break functionality).